src/sentry/static/sentry/app/views/organizationEvents/eventsTable.jsx

```diff
@@ -27,13 +27,16 @@ class EventsTableBody extends React.PureComponent {
     return events.map((event, eventIdx) => {
       const project = projectsMap.get(event.projectID);
       const trimmedMessage = event.message.split('\n')[0].substr(0, 100);
+
+      const eventLink = new Set(organization.features).has('sentry10')
+        ? `/organizations/`
```

Comment:
Won't this go to the issue dashboard now? Should they be sent to /organizations/:orgId/events instead?

Reply:
Oops, not sure what happened there - thanks